src/index.ts

```diff
@@ -62,6 +62,31 @@ class IconFontBuildr {
 
     this.config = mergeWith ( {}, this.configDefault, config, ( prev, next ) => isArray ( next ) ? next : undefined );
 
+    this.config.icons = Object.keys ( this.config.icons ).reduce( ( obj, val ) => {
```

Comment:
Calling `Object.keys` on an array feels weird, why not just calling reduce directly?

Reply:
You're right, I didn't notice that. Changing that should be fine 🙂